### PR TITLE
Update model names and notes to correspond to mmd 3.x models

### PR DESCRIPTION
**Pull request by PhaniShekhar**
# Description

# Checklist

- [x] I have read the contribution guidelines.
  - [General](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
  - [SDK](https://github.com/Azure/azureml-examples/blob/main/sdk/CONTRIBUTING.md)
  - [CLI](https://github.com/Azure/azureml-examples/blob/main/cli/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

**Comment by nvijayrania** [comment]
@PhaniShekhar:  I have a PR for updating azureml-examples https://github.com/Azure/azureml-examples/pull/2858/files and I think it covers the automl notebooks as well. Could you please check.

File: sdk/python/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/automl-image-instance-segmentation-task-fridge-items.ipynb [hunk 1]
```diff
@@ -1018,9 +1021,9 @@
    "source": [
     "### 4.3.1 Manual hyperparameter sweeping for models from MMDetection (Preview)\n",
     "\n",
-    "Similar to how you can use any model from MMDetection version 2.28.2 for individual runs, you can also include these models to perform a hyperparameter sweep. You can also choose a combination of models supported natively by [AutoML](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models?tabs=CLI-v2#configure-model-algorithms-and-hyperparameters) and models from [MMDetection](https://github.com/open-mmlab/mmdetection/blob/v2.28.2/docs/en/model_zoo.md).\n",
+    "Similar to how you can use any model from MMDetection for individual runs, you can also include these models to perform a hyperparameter sweep. You can also choose a combination of models supported natively by [AutoML](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models?tabs=CLI-v2#configure-model-algorithms-and-hyperparameters) and models from [MMDetection](https://github.com/open-mmlab/mmdetection/blob/v3.2.0/docs/en/model_zoo.md).\n",
```
Comment (rjaincc) [inline sdk/python/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/automl-image-instance-segmentation-task-fridge-items.ipynb:1024]:
```suggestion
    "Similar to how you can use any model from MMDetection for individual runs, you can also include these models to perform a hyperparameter sweep. You can also choose a combination of models supported natively by [AutoML](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models?tabs=CLI-v2#configure-model-algorithms-and-hyperparameters) and models from [MMDetection](https://github.com/open-mmlab/mmdetection/blob/v3.1.0/docs/en/model_zoo.md).\n",
```

File: sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb [hunk 2]
```diff
@@ -735,9 +735,17 @@
    "source": [
     "### 4.2.1 Individual runs with models from MMDetection (Preview)\n",
     "\n",
-    "In addition to the models supported natively by AutoML, you can launch individual runs to explore any model from MMDetection version 2.28.2 that supports object detection. Please refer to this [documentation](https://github.com/open-mmlab/mmdetection/blob/v2.28.2/docs/en/model_zoo.md) for the list of models.\n",
+    "In addition to the models supported natively by AutoML, you can launch individual runs to explore any model from MMDetection that supports object detection. Please refer to this [documentation](https://github.com/open-mmlab/mmdetection/blob/v3.2.0/docs/en/model_zoo.md) for the list of models.\n",
```
Comment (rjaincc) [inline sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb:738]:
```suggestion
    "In addition to the models supported natively by AutoML, you can launch individual runs to explore any model from MMDetection that supports object detection. Please refer to this [documentation](https://github.com/open-mmlab/mmdetection/blob/v3.1.0/docs/en/model_zoo.md) for the list of models.\n",
```

File: sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb [hunk 3]
```diff
@@ -1032,9 +1040,9 @@
    "source": [
     "### 4.3.1 Manual hyperparameter sweeping for models from MMDetection (Preview)\n",
     "\n",
-    "Similar to how you can use any model from MMDetection version 2.28.2 for individual runs, you can also include these models to perform a hyperparameter sweep. You can also choose a combination of models supported natively by [AutoML](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models?tabs=CLI-v2#configure-model-algorithms-and-hyperparameters) and models from [MMDetection](https://github.com/open-mmlab/mmdetection/blob/v2.28.2/docs/en/model_zoo.md).\n",
+    "Similar to how you can use any model from MMDetection for individual runs, you can also include these models to perform a hyperparameter sweep. You can also choose a combination of models supported natively by [AutoML](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models?tabs=CLI-v2#configure-model-algorithms-and-hyperparameters) and models from [MMDetection](https://github.com/open-mmlab/mmdetection/blob/v3.2.0/docs/en/model_zoo.md).\n",
```
Comment (rjaincc) [inline sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb:1043]:
```suggestion
    "Similar to how you can use any model from MMDetection for individual runs, you can also include these models to perform a hyperparameter sweep. You can also choose a combination of models supported natively by [AutoML](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-image-models?tabs=CLI-v2#configure-model-algorithms-and-hyperparameters) and models from [MMDetection](https://github.com/open-mmlab/mmdetection/blob/v3.1.0/docs/en/model_zoo.md).\n",
```

**Review by sdgilley — APPROVED** [review]
Docs approval

**Comment by PhaniShekhar** [comment]
> Instead of adding comments in the notebooks about some models being unsupported, why don't we modify the code snippet to only display models starting with mmd-3x?

Thanks for the suggestion. Made the corresponding changes.

**Comment by PhaniShekhar** [comment]
> @PhaniShekhar: I have a PR for updating azureml-examples https://github.com/Azure/azureml-examples/pull/2858/files and I think it covers the automl notebooks as well. Could you please check.

Ohh... I didn't realize that you had this PR out. Apologies. Is it okay if I leave this PR as is since there are a few minor additional changes and you exclude these sweep component files in your PR ?

**Comment by PhaniShekhar** [comment]
Changes are merged into Nilesh' PR here: https://github.com/Azure/azureml-examples/pull/2858

**Pull request closed without merging**